output/influxdb/config.go

```diff
@@ -141,40 +143,40 @@ func ParseURL(text string) (Config, error) {
 	if u.Host != "" {
 		c.Addr = null.StringFrom(u.Scheme + "://" + u.Host)
 	}
-	if db := strings.TrimPrefix(u.Path, "/"); db != "" {
-		c.DB = null.StringFrom(db)
-	}
 	if u.User != nil {
 		c.Username = null.StringFrom(u.User.Username())
 		pass, _ := u.User.Password()
 		c.Password = null.StringFrom(pass)
 	}
+	if bucket := strings.TrimPrefix(u.Path, "/"); bucket != "" {
+		c.Bucket = null.StringFrom(bucket)
+	}
 	for k, vs := range u.Query() {
 		switch k {
-		case "insecure":
+		case "insecureSkipTLSVerify":
```

Comment:
Please list this option rename as a breaking change ... and maybe we should support it for a verison or two and print a warning?